Realm/Realm/Logging/Logger.cs

```diff
@@ -64,39 +64,73 @@ public abstract class Logger
         /// <value>A <see cref="Logger"/> that doesn't output any messages.</value>
         public static Logger Null { get; } = new NullLogger();
 
+        /// <summary>
+        /// Gets a <see cref="FunctionLogger"/> that proxies Log calls to the supplied function. The message will
+        /// already be formatted with the default message formatting that includes a timestamp.
+        /// </summary>
+        /// <param name="logFunction">Function to proxy log calls to.</param>
+        /// <returns>
+        /// A <see cref="Logger"/> instance that will invoke <paramref name="logFunction"/> for each message.
+        /// </returns>
+        public static Logger Function(Action<string> logFunction) => new FunctionLogger((level, category, message) => logFunction(FormatLog(level, message, category)));
+
         /// <summary>
         /// Gets a <see cref="FunctionLogger"/> that proxies Log calls to the supplied function.
         /// </summary>
         /// <param name="logFunction">Function to proxy log calls to.</param>
         /// <returns>
         /// A <see cref="Logger"/> instance that will invoke <paramref name="logFunction"/> for each message.
         /// </returns>
-        public static Logger Function(Action<LogLevel, string> logFunction) => new FunctionLogger(logFunction);
+        [Obsolete("Use Function(Action<LogLevel, LogCategory, string> logFunction).")]
```

Comment:
Should we suggest the other new method too?

Reply:
There should only have bene one new function introduced here. Are you referring to `Function(Action<string> logFunction)`? If so, that one already existed but moved it up 👍

Reply:
My bad 😁